packages/discord.js/src/structures/Channel.js

```diff
@@ -260,6 +262,10 @@ class Channel extends Base {
             if (!allowUnknownGuild) channel.parent?.threads.cache.set(channel.id, channel);
             break;
           }
+          // TODO: Use discord-api-types
+          case 14:
```

Comment:
This should be available in next release, as https://github.com/discordjs/discord-api-types/pull/399 includes it. 👀

Reply:
Released